README.md

```diff
@@ -156,7 +156,7 @@ static let clientConfig = "" // <-- Change license here
         * zipped size: 3.8 MB
         * unzipped size: 9.3 MB
         * cache for neural networks and auxiliary files: 25 MB
-        * the size of cache for 3D models can be set up via API
+        * the size of cache for 3D models can be set up via API. (3D model size varies depending on it's 'quality. It is downloaded right after session starts.)
```

Comment:
`'quality`

What this ' means?

Reply:
same as in the next point - _RAM: up to 600 MB, depending on the size and quality of a 3D model_

Reply:
@nikita-wannafashion  what do you suggest for paraphrasing?

Reply:
I suggest converting this to a Note and removing the period at the end of the list item:
* the size of cache for 3D models can be set up via API<br />**Note:** 3D model size varies depending on its quality. The model will be downloaded right after session starts.
@Bliss00

Reply:
At least please change it's -> its.